File: MY APPLICATION 33/main/java/com/example/myapplication/SecondFragment.java
package com.example.myapplication;

import android.content.Intent;
import android.os.Bundle;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

public class SecondFragment extends Fragment
{

    TextView txt1;



    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view= inflater.inflate(R.layout.fragment_second, container, false);


        txt1=view.findViewById(R.id.txt1);

        txt1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {



                  /* Intent i =new Intent(getActivity(),MainActivity.class);
                    startActivity(i);*/


                FristFragment f1 =new FristFragment();
                FragmentManager fm=getFragmentManager();
                FragmentTransaction ft=fm.beginTransaction();
                ft.replace(R.id.frm1,f1).commit();

            }
        });


        return view;
    }
}

File: MY APPLICATION 19/main/java/com/example/myapplication/SecondFragment.kt
package com.example.materialex

import android.content.Intent
import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import com.example.myapplication.MainActivity
import com.example.myapplication.R


class SecondFragment : Fragment() {

    lateinit var txt1:TextView

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?, ): View?
    {
        // Inflate the layout for this fragment
        var view=inflater.inflate(R.layout.fragment_second, container, false)

        txt1=view.findViewById(R.id.txt2)
        txt1.setOnClickListener()
        {
            var i =Intent(activity,MainActivity::class.java)
            startActivity(i)
        }

        return view
    }


}